benchmarks/src/jmh/java/org/postgresql/benchmark/statement/ParseStatement.java

```diff
@@ -47,12 +47,12 @@
 @BenchmarkMode(Mode.AverageTime)
 @OutputTimeUnit(TimeUnit.MICROSECONDS)
 public class ParseStatement {
-  @Param({"0", "1", "10", "20"})
-  private int bindCount;
-
   @Param({"false"})
   public boolean unique;
-
+  @Param({"false", "true"})
+  public boolean named;
+  @Param({"0", "1", "10", "20", "100", "1000"})
```

Comment:
Using named placeholders doesn't seem to be hurting performance, at least not at 1000 placeholders.

Reply:
Do you mean "old code" performs exactly the same as the new one?

Could you please show the results?

Reply:
I would be happy to, as soon as the feature set stabilizes.